### ISSUE TITLE
No boot with Impossamole (1989)(Gremlin) 

### ISSUE THREAD

**Issue by Gianmarco72** [dirkwhoffmann/vAmiga#343]
After a few seconds vAmiga stops loading and black screen appear:
<img width="912" alt="Schermata 2020-05-16 alle 19 40 25" src="https://user-images.githubusercontent.com/45571833/82126480-5f2ad380-97ad-11ea-8276-c01b1291ad0e.png">

On UAE👍🏻:
![Paranoimia](https://user-images.githubusercontent.com/45571833/82126545-c6488800-97ad-11ea-8d57-a1da0936bf81.jpg)

**Comment by dirkwhoffmann** [comment]
Confirmed: Broken in vAmiga (and also in SAE).

**Comment by dirkwhoffmann** [comment]
For reference: Running FSUAE against vAmiga with this disk as boot disk: 

    ./fs-uae --kickstart-file=~/Desktop/Testing/kick13.rom  --uae_immediate_blits=true --floppy_drive_speed=0 --slow_memory=0 --floppy_drive_0=~/Desktop/Testing/Impossamole.adf > /tmp/uae.log

<img width="1425" alt="Bildschirmfoto 2020-05-21 um 18 30 35" src="https://user-images.githubusercontent.com/12561945/82581839-755fd780-9b91-11ea-8bb4-87e0124d5d47.png">

**Comment by dirkwhoffmann** [comment]
Findings: After 

     BLITTER Blit 357 (8,32) (1101) (0 0 78 0) 48de 4ade d378 1992 D
     BLITTER check1: bf6283fd check2: aeed93c5 ABCD: 46de 48de d378 1792

the games eventually switches off the power LED here: 

<img width="980" alt="Bildschirmfoto 2020-05-22 um 17 32 56" src="https://user-images.githubusercontent.com/12561945/82684316-abb15b80-9c52-11ea-82a3-c88eabd59aec.png">

Right after that, it branches to garbage code: 😳

<img width="980" alt="Bildschirmfoto 2020-05-22 um 17 33 37" src="https://user-images.githubusercontent.com/12561945/82684326-aeac4c00-9c52-11ea-8dcd-133cc6dfaf64.png">

The garbage here must be the result of an error that occurred way before.

**Comment by dirkwhoffmann** [comment]
I've come across the following mysterious behavior: When booting the game with SlowRam enabled, Blit 24 looks as follows in vAmiga:

```
BLITTER Blit 24 (40,256) (0001) (0 0 0 0) 291e 2b1e 0 60c8 
BLITTER check1: 5dacbdc5 check2: 64baadc5 ABCD: 291e 2b1e 0 b0c8
```

Just like as it looks in SAE/FSUAE/UAE: 

```
BLITTER Blit 24 (40,256) (0001) (0 0 0 0) 291e 2b1e 0 60c8 
BLITTER check1: 5dacbdc5 check2: 64baadc5 ABCD: 291e 2b1e 0 b0c8

```

When I turn off SlowRam in vAmiga, it changes to:

```
BLITTER Blit 24 (40,200) (0001) (0 0 0 0) 73ce 75ce 0 d520 
BLITTER check1: 157bb6c5 check2: b461b845 ABCD: 73ce 75ce 0 113a0
```

In SAE/FSUAE/UAE, it doesn't change. 🤔

**Comment by dirkwhoffmann** [comment]
This issue is very likely a duplicate of #348. Both issues can be tracked down to the same code. From the screenshots below, the first one is from Impossamole and the second one from The Jetsons: 

<img width="980" alt="impossamole2" src="https://user-images.githubusercontent.com/12561945/82990266-35b74680-9ffc-11ea-81b4-44d7aba0af26.png">

<img width="980" alt="Bildschirmfoto 2020-05-27 um 09 23 11" src="https://user-images.githubusercontent.com/12561945/82990271-37810a00-9ffc-11ea-9444-7719ab5ab091.png">

Maybe this code is from the Rob Northen Copylock (not sure yet).

**Comment by dirkwhoffmann** [comment]
Fixed in v0.9.8:

<img width="995" alt="Bildschirmfoto 2020-06-06 um 08 18 28" src="https://user-images.githubusercontent.com/12561945/83937639-e1b41b00-a7ce-11ea-9774-4d0c42674315.png">

<img width="995" alt="Bildschirmfoto 2020-06-06 um 08 18 46" src="https://user-images.githubusercontent.com/12561945/83937640-e4167500-a7ce-11ea-8446-db9e30ba2ec4.png">
